pkg/runtime/config.go

```diff
@@ -14,47 +14,75 @@
 package runtime
 
 import (
+	"errors"
```

Comment:
nit: a space line after this line to separate standard import from external ones